hack/arktos-cni.rc

```diff
@@ -230,14 +245,17 @@ EOF"
 
 CNIPLUGIN=${CNIPLUGIN:-"bridge"}
 cni_conf_dir=${CNI_CONF_DIR:-"/etc/cni/net.d"}
+cni_conf_parent_dir=${CNI_CONF_PARENT_DIR:-"/etc/cni"}
 cni_bin_dir=${CNI_BIN_DIR:-"/opt/cni/bin"}
 
```

Comment:
these vars are not global, only use in the file hack/arktos-cni.rc. The GLOBAL vars are defined in the file hack/lib/common-var-init.sh.

Reply:
The variables with lower case should be used in the file hack/arktos-cni.rc ONLY whereas the variables with upper case should be GLOBAL variables, which may be used by other scripts because thhe hack/arktos-cni.rc are called by other scripts.

Reply:
The new codes achieve this goal - the local variables are lower-case and the global variables are upper-case in hack/arktos-cni.rc